apps/editing-toolkit/editing-toolkit-plugin/global-styles/README.md

```diff
@@ -45,7 +45,7 @@ h1 {
 
 In the above example, the `body`'s font-family will be what the user selected as the base font, or `serif` if `--font-base` is not set. Likewise, the `h1`'s font-family will be what the user selected as the headings font, or `sans-serif` if `--font-headings` is not set.
 
-**Fallback values for CSS Custom Properties**
+### Fallback values for CSS Custom Properties
```

Comment:
I'm noticing that in some cases we replace bold instances with h3, sometimes with h2, and sometimes with h4. Is there any fixed rule that we're following when we're doing that replacement?

Reply:
What I did was look at other headers in the doc and figure out which level would make more sense. In some cases it was obvious what the right choice was, but in others it was a bit subjective.

Reply:
No harm done I guess. Not-the-right header is likely better than no header at all 😉